src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -199,12 +199,17 @@ public void Abort()
                 ConnectionControl.End(ProduceEndType.SocketDisconnect);
                 SocketInput.AbortAwaiting();
 
-                _disconnectCts.Cancel();
+                _abortedCts?.Cancel();
```

Comment:
Maybe wrap this call specifically since we're not locking to avoid ODEs (however unlikely). This prevents us from logging errors that we know can happen, but the app developer can do nothing about.

        try
        {
            _abortedCts.Cancel();
        }
        catch (ObjectDisposedException)
        {
            // Don't log ODEs thrown from _abortedCts.Cancel()
            // If _disconnectCts is disposed, the app has already completed.
        }